.github/workflows/libc-fullbuild-tests.yml

```diff
@@ -11,12 +11,18 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-24.04
+    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
         include:
-          - c_compiler: clang
+          - os: ubuntu-24.04
+            ccache-variant: sccache
```

Comment:
Why use `sccache` for this, but `ccache` for the arm variant?

Reply:
`sccache` is a modern implementation from Mozilla, it supports more storage format and compiler (e.g. `msvc/clang-cl`). I prefer to default on `sccache` but due to https://github.com/hendrikmuhs/ccache-action/issues/279, `ccache` is used on arm64 ubuntu.

Reply:
Please add a comment linking to that issue in the yml file. I assume it will be resolved some day, so if someone notices that in the future, it would be good to then resolve the arm specific differences.

Reply:
Addressed